recipe/build.sh

```diff
@@ -9,8 +9,16 @@
 #
 source activate "${CONDA_DEFAULT_ENV}"
 
+autoreconf --force --install
 ./autogen.sh
-./configure --with-pic --prefix=$PREFIX
+if [ "$(uname)" == "Linux" ]; then
+  export CXXFLAGS="${CXXFLAGS} -O2 -DNDEBUG"
+else
+  # There is at least on OSX a bug affecting -O2 builds
+  export CXXFLAGS="${CXXFLAGS} -O1 -DNDEBUG"
```

Comment:
Already raised upstream, as they don't seem to have a bug tracker, posted to the ML: https://groups.google.com/forum/#!topic/snappy-compression/TD-x9gt6mjw Sadly no response yet.